src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/HeaderFileBuilder.java

```diff
@@ -94,7 +94,7 @@ public void addFunction(String javaName, String nativeName, FunctionInfo functio
             emitFunctionWrapper(mhConstant, javaName, nativeName, functionInfo);
             if (functionInfo.methodType().returnType().equals(MemorySegment.class)) {
                 // emit scoped overload
-                emitFunctionWrapperNoAllocatorOverload(javaName, functionInfo);
+                // emitFunctionWrapperNoAllocatorOverload(javaName, functionInfo);
```

Comment:
Any reason as to why this is commented as opposed to removed (along with the corresponding, now unused, method) ?